app/upw/templates/pdf-preview-and-declaration/components/pdf-template/template.njk

```diff
@@ -752,22 +752,16 @@
 
     <div class='upw-avoid-break'>
         <h2 class="govuk-heading-m">How long will it take for the individual to complete their hours?</h2>
-
+        <p>Figures are approximate and based on the individual completing 7 hours of work each day.</p>
+        <br>
         <table class="upw-pdf-chart">
             <tbody>
                 <tr>
-                    <td class="upw-pdf-chart__cell--empty"></td>
-                    <td class="upw-pdf-chart__cell--light-grey upw-pdf-chart__cell--border-top" colspan="4">If the individual works…</td>
-                </tr>
-                <tr>
-                    <th class="upw-pdf-chart__cell--mid-grey upw-pdf-chart__cell--border-top upw-pdf-chart__cell--border-right" rowspan="2">Number of hours to complete</th>
-                    <th class="upw-pdf-chart__cell--mid-grey upw-pdf-chart__cell--border-top upw-pdf-chart__cell--border-right">1 day per week</th>
-                    <th class="upw-pdf-chart__cell--mid-grey upw-pdf-chart__cell--border-top upw-pdf-chart__cell--border-right">2 days per week</th>
-                    <th class="upw-pdf-chart__cell--mid-grey upw-pdf-chart__cell--border-top upw-pdf-chart__cell--border-right">3 days per week</th>
-                    <th class="upw-pdf-chart__cell--mid-grey upw-pdf-chart__cell--border-top">4 days per week</th>
-                </tr>
-                <tr>
-                    <td class="upw-pdf-chart__cell--light-grey upw-pdf-chart__cell--border-top" colspan="4">…they will complete their hours in this many weeks*</td>
+                    <th class="upw-pdf-chart__cell--mid-grey upw-pdf-chart__cell--border-top upw-pdf-chart__cell--border-right">Number of hours of community payback to complete</th>
```

Comment:
This table heading doesn't have paragraph tags around it's content, just wanting to check this is intentional?

Reply:
Yes it was intentional, following the figma designs where this heading is to stay in bold but the other headings only certain text was to be in bold.

Reply:
👍